Library/Formula/libu2f-server.rb

```diff
@@ -0,0 +1,26 @@
+class Libu2fServer < Formula
+  desc "Yubico Universal 2nd Factor (U2F) Server C Library"
+  homepage "https://developers.yubico.com/libu2f-server/"
+  url "https://developers.yubico.com/libu2f-server/Releases/libu2f-server-1.0.1.tar.xz"
+  sha256 "a618f59051209d6d70c24cf42d64c9b67bd7dd5946b6dbd2c649181d7e8f1f6e"
+
+  depends_on "check" => :build
+  depends_on "json-c"
+  depends_on "openssl"
+
+  def install
+    args = ["--prefix=#{prefix}"]
+    ENV["LIBSSL_LIBS"] = "-lssl -lcrypto -lz"
+    ENV["LIBCRYPTO_LIBS"] = "-lcrypto -lz"
+
+    ENV["HELP2MAN"] = "/usr/local/bin/help2man --no-discard-stderr"
```

Comment:
Presumably you need a dependency for this?

Also: Homebrew isn't always in `/usr/local`; as much as we encourage people to put it there you can actually put it anywhere you like.

Also: `HOMEBREW_PREFIX/bin` won't be in the `$PATH` during compile. You should call direct formula installations, like this:

```ruby
ENV["HELP2MAN"] = "#{Formula["help2man"].opt_bin}/help2man", "--no-discard-stderr"
```